### PR TITLE
Calculator Component

### PR DESCRIPTION
**Pull request by HansZizold**
Calculator component added
Styles were added to match the design.
Readme file updated.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
```
Comment (rotimiazeez) [inline src/App.js]: - Kindly remove any unnecessary lines of code to make your code base good and neat 😉

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,14 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main>
+      <section>
+        <Calculator />
+      </section>
+    </main>
```
Comment (rotimiazeez) [inline src/App.js]: Please make sure your `App.js` file uses a class-based component as required 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @HansZizold 👏🏽 , 
## Good job so far! 👍🏽
### Summary 💯
- No linter error ✔️
- Good PR description 👍🏽
- Correct git-flow followed 💯 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (HansZizold) [reply src/App.js]: Hi @rotimiazeez, thank you for your observation. I thought that classes were needed only for the component. But now I have also included them in the app.js file.

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @HansZizold,

Good job implementing the changes asked by the previous reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App components use class-based components

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**